core/model_induction.py

```diff
@@ -170,12 +170,17 @@ def dump(self, file_path):
             file.write(file_buffer)
 
 
-def train_decision_tree(train_features, train_label):
+def train_custom_tree(train_features, train_label):
     model = CustomBinaryClassifier()
     model.fit(train_features, train_label)
     return model
 
-def train_classifier_tree(train_features, train_label):
-    model = RandomForestClassifier(n_estimators=30, max_depth=40)
-    model.fit(*balance_binary_dataset(train_features, train_label, skew_false=8))
+def train_decision_tree(train_features, train_label):
+    model = DecisionTreeClassifier()
+    model.fit(train_features, train_label)
+    return model
+
+def train_random_forest(train_features, train_label, n_estimators=30, max_depth=40, **kwargs):
+    model = RandomForestClassifier(n_estimators, max_depth, **kwargs)
```

Comment:
sorry lol last thing, is this tested? if the initializer complains about args order u might need to specify these as kwargs

Reply:
Yeah, just fixed now.